responses.json

```diff
@@ -183,5 +183,21 @@
                 }
             }
         }
+    },
+    "ReferenceHash": {
+        "description": "Modified Reference Hash",
+            "content": {
+                "application/json": {
+                    "schema": { "$ref": "./schemas.json#/Fingerprint" }
```

Comment:
so the hash itself is not a `Fingerprint` per se. it's really just a string computed from the fingerprint